### PR TITLE
latex support for plantuml

### PR DESCRIPTION
**Pull request by mollaf**
I want to use latex notation in Plantuml so I made some changes in your code. In my environment it is working, I would be glad if you check and merge. I downloaded the binaries required to use latex in Plantuml from "https://plantuml.com/ascii-math". I had to make some changes to the code as the svg result from Plantuml could not return in one piece. The Plantuml code and the return result are below.

```plantuml
@startuml

rectangle fx [
    <math>f(x)</math>
]

@enduml
```

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentScriptType="application/ecmascript" contentStyleType="text/css" height="66px" preserveAspectRatio="none" style="width:87px;height:66px;" version="1.1" viewBox="0 0 87 66" width="87px" zoomAndPan="magnify"><defs><filter height="300%" id="fp9by2kzp41g0" width="300%" x="-1" y="-1"><feGaussianBlur result="blurOut" stdDeviation="2.0"/><feColorMatrix in="blurOut" result="blurOut2" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 .4 0"/><feOffset dx="4.0" dy="4.0" in="blurOut2" result="blurOut3"/><feBlend in="SourceGraphic" in2="blurOut3" mode="normal"/></filter></defs><g><!--MD5=[b33f32fa337413e907b452d2aca0ec76]
entity fx--><rect fill="#FEFECE" filter="url(#fp9by2kzp41g0)" height="48" style="stroke: #000000; stroke-width: 1.5;" width="69" x="7" y="7"/><svg x="17" y="17"><!--Generated by the Batik Graphics2D SVG Generator--><defs id="genericDefs"
  /><g
  ><g transform="scale(20,20) translate(0.2,0.95)" style="font-size:1; text-rendering:geometricPrecision; color-rendering:optimizeQuality; image-rendering:optimizeQuality; font-family:&apos;jlm_cmmi10&apos;; color-interpolation:linearRGB;"
    ><path style="stroke:none;" d="M0.3594 -0.4062 L0.3594 -0.4062 L0.3125 -0.0781 Q0.2969 -0.0156 0.2656 0.0625 Q0.2656 0.0938 0.25 0.1094 Q0.2188 0.1875 0.1562 0.2031 Q0.1562 0.2031 0.1406 0.2031 Q0.0938 0.2031 0.0625 0.1719 Q0.0469 0.1562 0.0469 0.1406 Q0.0469 0.0938 0.0938 0.0781 Q0.0938 0.0781 0.1094 0.0781 Q0.1406 0.0781 0.1406 0.0938 Q0.1406 0.1094 0.1406 0.1094 Q0.1406 0.1406 0.1094 0.1562 Q0.1094 0.1562 0.0938 0.1719 Q0.1094 0.1875 0.1406 0.1875 Q0.1875 0.1875 0.2031 0.0781 Q0.2188 0.0625 0.2188 0.0312 L0.2188 0 L0.2969 -0.4062 L0.2344 -0.4062 Q0.2031 -0.4062 0.2031 -0.4062 L0.2031 -0.4062 L0.2031 -0.4062 Q0.2031 -0.4219 0.2188 -0.4375 L0.2188 -0.4375 L0.2344 -0.4375 L0.3125 -0.4375 Q0.3281 -0.5625 0.3438 -0.5938 Q0.375 -0.6875 0.4375 -0.7031 Q0.4531 -0.7031 0.4688 -0.7031 Q0.5156 -0.7031 0.5469 -0.6719 L0.5469 -0.6719 Q0.5469 -0.6562 0.5469 -0.6406 Q0.5469 -0.5938 0.5156 -0.5781 Q0.5156 -0.5781 0.5 -0.5781 Q0.4688 -0.5781 0.4688 -0.5938 Q0.4531 -0.6094 0.4531 -0.6094 Q0.4531 -0.6406 0.4844 -0.6562 Q0.5 -0.6562 0.5156 -0.6719 Q0.5 -0.6875 0.4688 -0.6875 Q0.4375 -0.6875 0.4219 -0.6562 Q0.4062 -0.6406 0.3906 -0.5469 L0.375 -0.4375 L0.4531 -0.4375 Q0.4844 -0.4375 0.4844 -0.4219 Q0.4844 -0.4062 0.4688 -0.4062 L0.4688 -0.4062 L0.4531 -0.4062 L0.3594 -0.4062 Z"
    /></g
    ><g transform="matrix(20,0,0,20,0,0) translate(0.8563,0.95)" style="font-size:1; text-rendering:geometricPrecision; color-rendering:optimizeQuality; image-rendering:optimizeQuality; font-family:&apos;jlm_cmr10&apos;; color-interpolation:linearRGB;"
    ><path style="stroke:none;" d="M0.3281 0.2344 L0.3281 0.2344 Q0.3281 0.25 0.3281 0.25 Q0.3125 0.25 0.2656 0.2031 Q0.2031 0.1406 0.1719 0.0625 Q0.0938 -0.0781 0.0938 -0.25 Q0.0938 -0.4219 0.1562 -0.5625 Q0.2188 -0.6719 0.2969 -0.7344 Q0.3125 -0.75 0.3281 -0.75 Q0.3281 -0.75 0.3281 -0.7344 Q0.3281 -0.7344 0.3125 -0.7188 Q0.1562 -0.5625 0.1562 -0.25 Q0.1562 0.0625 0.3125 0.2188 Q0.3281 0.2344 0.3281 0.2344 Z"
    /></g
    ><g transform="matrix(20,0,0,20,0,0) translate(1.2451,0.95)" style="font-size:1; text-rendering:geometricPrecision; color-rendering:optimizeQuality; image-rendering:optimizeQuality; font-family:&apos;jlm_cmmi10&apos;; color-interpolation:linearRGB;"
    ><path style="stroke:none;" d="M0.3281 -0.2969 L0.2969 -0.1719 Q0.2812 -0.0938 0.2812 -0.0781 Q0.2812 -0.0312 0.3125 -0.0156 Q0.3281 -0.0156 0.3438 -0.0156 Q0.3906 -0.0156 0.4219 -0.0625 Q0.4531 -0.0938 0.4688 -0.1406 Q0.4688 -0.1562 0.4844 -0.1562 Q0.5 -0.1562 0.5 -0.1406 Q0.5 -0.125 0.4688 -0.0781 Q0.4062 0.0156 0.3438 0.0156 Q0.2656 0.0156 0.2344 -0.0469 Q0.2344 -0.0625 0.2344 -0.0625 Q0.2031 -0.0156 0.1562 0 Q0.1406 0.0156 0.125 0.0156 Q0.0625 0.0156 0.0312 -0.0312 Q0.0312 -0.0312 0.0312 -0.0625 Q0.0312 -0.0938 0.0625 -0.1094 Q0.0781 -0.1094 0.0781 -0.1094 Q0.1094 -0.1094 0.125 -0.0938 Q0.125 -0.0781 0.125 -0.0781 Q0.125 -0.0469 0.0938 -0.0312 Q0.0781 -0.0312 0.0781 -0.0312 Q0.0938 -0.0156 0.125 -0.0156 Q0.1719 -0.0156 0.2031 -0.0781 Q0.2188 -0.0938 0.2188 -0.1094 Q0.2656 -0.3125 0.2656 -0.3594 Q0.2656 -0.4219 0.2188 -0.4219 Q0.1719 -0.4219 0.125 -0.3594 L0.125 -0.3594 Q0.125 -0.3594 0.125 -0.3594 Q0.1094 -0.3438 0.0938 -0.3125 Q0.0938 -0.3125 0.0938 -0.2969 L0.0781 -0.2812 Q0.0781 -0.2812 0.0781 -0.2812 Q0.0625 -0.2812 0.0625 -0.2812 Q0.0625 -0.3125 0.0938 -0.3594 Q0.1406 -0.4375 0.2188 -0.4375 Q0.2969 -0.4375 0.3281 -0.3594 Q0.3594 -0.4219 0.4062 -0.4375 Q0.4219 -0.4375 0.4375 -0.4375 Q0.5 -0.4375 0.5156 -0.4062 Q0.5312 -0.3906 0.5312 -0.375 Q0.5312 -0.3281 0.4844 -0.3281 Q0.4844 -0.3125 0.4688 -0.3125 Q0.4375 -0.3125 0.4375 -0.3438 Q0.4375 -0.3438 0.4375 -0.3594 Q0.4375 -0.3906 0.4688 -0.4062 Q0.4688 -0.4062 0.4844 -0.4062 Q0.4688 -0.4219 0.4375 -0.4219 Q0.3594 -0.4219 0.3438 -0.3125 Q0.3281 -0.3125 0.3281 -0.2969 Z"
    /></g
    ><g transform="matrix(20,0,0,20,0,0) translate(1.8167,0.95)" style="font-size:1; text-rendering:geometricPrecision; color-rendering:optimizeQuality; image-rendering:optimizeQuality; font-family:&apos;jlm_cmr10&apos;; color-interpolation:linearRGB;"
    ><path style="stroke:none;" d="M0.2812 -0.25 L0.2812 -0.25 Q0.2812 -0.0781 0.2188 0.0625 Q0.1719 0.1719 0.0938 0.2344 Q0.0781 0.25 0.0625 0.25 Q0.0625 0.25 0.0625 0.2344 Q0.0625 0.2344 0.0625 0.2188 Q0.2344 0.0625 0.2344 -0.25 Q0.2344 -0.5625 0.0781 -0.7188 L0.0781 -0.7188 Q0.0625 -0.7344 0.0625 -0.7344 Q0.0625 -0.75 0.0625 -0.75 Q0.0781 -0.75 0.125 -0.7031 Q0.1875 -0.6406 0.2188 -0.5625 Q0.2812 -0.4219 0.2812 -0.25 Z"
    /></g
  ></g
></svg
>
<!--MD5=[3d7f9515c60dc76f35e46a19f2abbb87]
@startuml

rectangle fx [
    <math>f(x)</math>
]

@enduml

PlantUML version 1.2020.10(Sun May 17 12:35:57 TRT 2020)
(MIT source distribution)
Java Runtime: OpenJDK Runtime Environment
JVM: OpenJDK 64-Bit Server VM
Java Version: 11.0.7+10-post-Ubuntu-2ubuntu218.04
Operating System: Linux
Default Encoding: UTF-8
Language: en
Country: US
--></g></svg>
```

### CODE REVIEW

File: src/puml.ts [hunk 1]
```diff
@@ -60,13 +60,17 @@ class PlantUMLTask {
       "UTF-8",
     ]);
 
+    var svgData = '';
```
Comment (shd101wyy) [inline src/puml.ts:63]: I would suggest to use `let` instead of `var`

Comment (shd101wyy) [reply src/puml.ts:63]: Also, the TravisCI failed. It would be great if you could run prettier. Thank you!

Comment (rickardholmberg) [reply src/puml.ts:63]: Also, what is the reason behind keeping `svgData` separate from `this.chunks`? It is still `this.chunks` that is used to parse the diagrams below.

**Comment by shd101wyy** [comment]
Sorry out of date

**Pull request closed without merging**